core/common/src/MapLayerSettings.ts

```diff
@@ -112,35 +116,75 @@ export class MapSubLayerSettings {
   public get idString(): string { return (typeof this.id === "number") ? this.id.toString(10) : this.id; }
 }
 
-/** JSON representation of the settings associated with a map layer.  One or more map layers may be included within a [[MapImageryProps]] object.
+/** JSON representation of properties common to both [[ImageMapLayerProps]] and [[ModelMapLayerProps]].
  * @see [[MapImageryProps]]
  * @beta
  */
-export interface MapLayerProps {
+export interface CommonMapLayerProps {
   /** Controls visibility of layer. Defaults to 'true'. */
   visible?: boolean;
-  /** Identifies the map layers source.*/
-  formatId: string;
-  /** Name */
+
+  /** A user-friendly name for the layer. */
   name: string;
+  /** A transparency value from 0.0 (fully opaque) to 1.0 (fully transparent) to apply to map graphics when drawing,
+   * or false to indicate the transparency should not be overridden.
+   * Default value: 0.
+   */
+  transparency?: number;
+  /** True to indicate background is transparent.
+   * Default: true.
+   */
+  transparentBackground?: boolean;
+}
+
+/** JSON representation of an [[ImageMapLayer]].
+ * @see [[MapImagerySettings]].
+ * @beta
+ */
+export interface ImageMapLayerProps extends CommonMapLayerProps {
   /** URL */
   url: string;
+  /** Identifies the map layers source.*/
+  formatId: string;
   /** Source layers. If undefined all layers are displayed. */
   subLayers?: MapSubLayerProps[];
-  /** A transparency value from 0.0 (fully opaque) to 1.0 (fully transparent) to apply to map graphics when drawing,
-   * or false to indicate the transparency should not be overridden. Default value: 0.
-   * If omitted, defaults to 0. */
-  transparency?: number;
-  /** True to indicate background is transparent.  Defaults to 'true'. */
-  transparentBackground?: boolean;
-  /** Is a base layer.  Defaults to 'false'. */
-  isBase?: boolean;
   /** Access Key for the Layer, like a subscription key or access token.
-   * TODO This does not belong in the props object. It should never be persisted.
+   * ###TODO This does not belong in the props object. It should never be persisted.
    */
   accessKey?: MapLayerKey;
+  /** Is a base layer.  Defaults to 'false'. */
+  isBase?: boolean;
```

Comment:
@RBBentley @mdastous-bentley the meaning of this 'isBase` property is unclear to me. A MapImageryProps can have at most one base layer. But you can set isBase to true or false for any ImageMapLayer, including a base layer.
It seems to me that whether or not a given layer is a base layer should be determined entirely by whether or not it is assigned to MapImageryProps.backgroundBase, not a JSON property of the layer itself.

Reply:
I have removed the `isBase` property. Please resolve this comment unless you object.